CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 2.4.1 (05.02.2021)
+
+Added Troubleshooting section to [README.md](README.md).
+
+Added error handler for `jpegoptim` child_process.
```

Comment:
> child process

or

`child_process`

Otherwise it looks like a piece of code that wasn't wrapped with backticks.